pkg/services/ngalert/store/alert_rule.go

```diff
@@ -397,36 +397,19 @@ func (st DBstore) GetNamespaceByUID(ctx context.Context, uid string, orgID int64
 	return folder, nil
 }
 
-func (st DBstore) getFilterByOrgsString() (string, []interface{}) {
```

Comment:
The diff on the test is also borked - but this essentially is the meat of the diff, I got rid of this function in favour of using the ORM.

Reply:
I would recommend keeping this because we're migrating away from XORM to the `github.com/jmoiron/sqlx`

Reply:
The [sqlx migration has been paused](https://raintank-corp.slack.com/archives/CMSUYD6H3/p1680890445589139) for the time being - I have cleared this PR with the Grafana backend team.